bramkouwen/Music-Mixer
# updating js file name

## What does the PR do?
* It updates the js file name

## How do you check this PR?
* Pull down the branch
*  See changed js file name

## Screenshots 
None.
